client/asset/xmr/process.go

```diff
@@ -157,7 +186,7 @@ func cliGenerateRefreshWallet(ctx context.Context, trustedDaemon string, net dex
 	cmd.Args = append(cmd.Args, CliGenFromSpendKeyParam)
 	walletFilePath := path.Join(dataDir, WalletFileName)
 	cmd.Args = append(cmd.Args, walletFilePath)
-	cmd.Args = append(cmd.Args, CliPasswordParam+hex.EncodeToString(pw))
+	cmd.Args = append(cmd.Args, CliPasswordParam+string(pw))
```

Comment:
Encoded as hex is better, there could be unreadable symbols in utf-8. This is a hash I believe.